src/Backend/Version.ts

```diff
@@ -20,15 +20,22 @@ class Version {
   patch: number;
   option: string;
 
-  constructor(major: number = 0, minor: number = 0, patch: number = 0, option: string = '') {
+  constructor(major: number = 0, minor: number = -1, patch: number = -1, option: string = '') {
```

Comment:
(optional) I prefer `undefined` instead of `-1` in typescript but `-1` is also good to me.